### PR TITLE
Feat: RDCC-2785 - Create flyway scripts for inserting data into the static tables

### PR DESCRIPTION
**Pull request by AdilOoze**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RDCC-2785
&
https://tools.hmcts.net/jira/browse/RDCC-2929

### Change description ###

Create flyway scripts for inserting data into the static tables

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by SujeethKumarAri — CHANGES_REQUESTED** [review]
Can't we use something like this
`INSERT INTO table_name (column_list)
VALUES
    (value_list_1),
    (value_list_2),
    ...
    (value_list_n);` instead of separate values being assigned

**Review by SujeethKumarAri — APPROVED** [review]

File: src/main/resources/db/migration/V1_10__static_table_inserts.sql [hunk 1]
```diff
@@ -0,0 +1,343 @@
+insert into court_location_category(
+    court_location_category_id,
+    court_location_category,
+    welsh_court_location_category,
+    created_time
+    )
+values(
+    '1',
+    'Admin Court',
+    ' ',
```
Comment (ishratt) [inline src/main/resources/db/migration/V1_10__static_table_inserts.sql:10]: we're using `' '` because the field cannot be null?

Comment (AdilOoze) [reply src/main/resources/db/migration/V1_10__static_table_inserts.sql:10]: Yes @ishratt - this is what @kavithagade12 asked me to do:

> _"non-mandatory column keept it blank"_

Comment (kavithagade12) [reply src/main/resources/db/migration/V1_10__static_table_inserts.sql:10]: All the columns in the Data model which starts with welsh are blank. and they are optional(non mandatory).

Comment (ishratt) [reply src/main/resources/db/migration/V1_10__static_table_inserts.sql:10]: Thanks for the comments, @kavithagade12 please correct me if I'm wrong here.. according to the flyway script `V1_9__create_tables.sql` this is what we have in the `court_location_category` table: `welsh_court_location_category varchar(128) NOT NULL` doesn't this mean this field cannot be null, hence mandatory? All the other columns starting with welsh seem to be optional as you said except for this one.

Comment (kavithagade12) [reply src/main/resources/db/migration/V1_10__static_table_inserts.sql:10]: As mentioned all the  fields which starts with welsh_*.* are non mandatory. Please change  welsh_court_location_category to nullable field.

Comment (kavithagade12) [reply src/main/resources/db/migration/V1_10__static_table_inserts.sql:10]: The following value should be null rather than '  ',

**Pull request closed without merging**